src/nvim/api/buffer.c

```diff
@@ -126,6 +126,7 @@ String buffer_get_line(Buffer buffer, Integer index, Error *err)
 /// @param  opts  Optional parameters.
 ///             - on_lines: Lua callback invoked on change.
 ///               Return `true` to detach. Args:
+///               - "lines"
```

Comment:
https://github.com/neovim/neovim/blob/f5c56f03bb9ee25c3d931034497dc76a5591b770/src/nvim/buffer_updates.c#L240

Reply:
just "lines" can be confusing: `the string "lines"`